consutil/lib.py

```diff
@@ -37,6 +43,19 @@
 DEV_READY_MSG = r"([Ll]ogin:|[Pp]assword:|[$>#])" # login prompt or command line prompt
 TIMEOUT_SEC = 0.2
 
+
+proc = subprocess.Popen(PLATFORM,stdout=subprocess.PIPE,shell=True,stderr=subprocess.STDOUT)
```

Comment:
We might able to get the platform plugin path by using existing api:
https://github.com/Azure/sonic-utilities/blob/fdda5aca66af8ebcc9a8ab597851f134090b61a0/ssdutil/main.py#L35

Reply:
Made change followed this sample.